src/App.jsx

```diff
@@ -14,45 +16,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY_VALUE = 'Sort alphabetically';
+const SORT_BY_LENGTH_VALUE = 'Sort by length';
+const RESET_VALUE = 'Reset';
+const REVERSE_VALUE = 'Reverse';
+const REVERSE_DEFAULT_VALUE = false;
```

Comment:
Also, it's too much to create a constant for a boolean value and then compare in `getGoodsSorted`, just enough to have a flag `isReversed` in the component and use it.

And one more moment, it's better to name `getSortedGoods` or even `getPreparedGoods` as we also reverse them in some cases.